.vsts-ci.yml

```diff
@@ -267,6 +269,7 @@ jobs:
         TASK: swig
       cpp_tests:
         TASK: cpp-tests
+  timeoutInMinutes: 480
```

Comment:
We are using Microsoft-hosted agent for Windows jobs, so this value cannot be bigger than `360` in our case.
> For 360 minutes (6 hours) on Microsoft-hosted agents with a public project and public repository
    https://docs.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#timeouts